src/Model/PaymentDetails.php

```diff
@@ -6,6 +6,14 @@
 
 class PaymentDetails
 {
+    public const CARD_TYPE = 'card_type';
```

Comment:
Can this be like this, or maybe should I create a trait f.e. TpayGatewayType and store this values there?